packages/server/src/sdk/app/backups/imports.ts

```diff
@@ -151,7 +151,8 @@ export function getListOfAppsInMulti(tmpPath: string) {
 export async function importApp(
   appId: string,
   db: Database,
-  template: TemplateType
+  template: TemplateType,
+  opts: { objStore: boolean } = { objStore: true }
```

Comment:
What is this `objStore`?

Reply:
Its the object store, basically means whether it should try to import anything to the object store or not, theres a similar parameter on the export - I can rename them.